diff --git a/include/SKSE/Version.h b/include/SKSE/Version.h
index 633dfde..f3936ae 100644
--- a/include/SKSE/Version.h
+++ b/include/SKSE/Version.h
@@ -28,8 +28,9 @@ namespace SKSE
     constexpr REL::Version RUNTIME_SSE_1_6_640(1, 6, 640, 0);
 	constexpr REL::Version RUNTIME_SSE_1_6_659(1, 6, 659, 0);
 	constexpr REL::Version RUNTIME_SSE_1_6_678(1, 6, 678, 0);
-	constexpr REL::Version RUNTIME_SSE_1_6_1330(1, 5, 1330, 0);
-	constexpr auto         RUNTIME_SSE_LATEST_AE = RUNTIME_SSE_1_6_1330;
+	constexpr REL::Version RUNTIME_SSE_1_6_1130(1, 6, 1130, 0);
+	constexpr REL::Version RUNTIME_SSE_1_6_1170(1, 6, 1170, 0);
+	constexpr auto         RUNTIME_SSE_LATEST_AE = RUNTIME_SSE_1_6_1170;
 	constexpr auto         RUNTIME_SSE_LATEST_SE = RUNTIME_SSE_1_5_97;
 	constexpr auto         RUNTIME_SSE_LATEST = RUNTIME_SSE_LATEST_AE;
 
